### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by rasummer**
Updates the Readme inside docs source. Adds examples of inline text that respects supergroup preferences such as Java/Scala

### CODE REVIEW

**Review by octonato — APPROVED** [review]
LGTM

**Comment by octonato** [comment]
closing and reopening to retrigger Travis CI

**Comment by octonato** [comment]
anyway, this is just a README file change, we don't need Travis CI. But in any case, Travis seems to be stuck

File: docs-source/README.adoc [hunk 1]
```diff
@@ -111,7 +111,38 @@ Follow these steps to install Virtualbox on Ubuntu 16.04+:
...
+== Inline text exammples that respect Supergroups
+Inline text will respect a supergroup preference such as Java/Scala when using the following syntax:
+
+=== Plain Text
+
+[source, asciidoc]
+----
+Plain inline text about [.group-scala]#scala# [.group-java]#java#.
+
+Bold styling [.group-scala]#*scala*# [.group-java]#*java*# or formatted as [.group-scala]#`scala code`# [.group-java]#`java code`#.
```
Comment (patriknw) [inline docs-source/README.adoc:123]: great!

**Pull request merged**